### DISCUSSION TITLE
Having input fields inside multiSelect box

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#321] [PrimeVue]
How can I have input fields inside multiselect box? For example I want to have 2 input fields inside select box for price or year.
Is it possible to adjust multiSelect box for it and if yes, how?

**Comment by tugcekucukoglu** [comment]
You could use the header slot, but it will not give the expected result in the UI. That's why you can create or extend your own MultiSelect component.

**Reply by Uzeyir9695** [reply]
Okay, thank you ( Teshekkurler )) ).
